extensions/objects/templates/custom-osk-letters-editor.component.ts

```diff
@@ -82,12 +82,12 @@ export class CustomOskLettersEditorComponent implements OnInit {
 
   getRemainingLettersCount(): number {
     return Math.max(
-      AppConstants.MAX_CUSTOM_LETTERS_FOR_OSK - this.value.length, 0);
+      AppConstants.MAX_CUSTOM_LETTERS_FOR_OSK - this.value?.length, 0);
```

Comment:
Why is the '?' needed? Can we initialize it properly so that it is always valid, instead?

Reply:
It changes through the life cycle of the component. Don't matter if we initialise it well.